src/server_manager/web_app/app.ts

```diff
@@ -831,37 +855,49 @@ export class App {
     }
   }
 
-  private showTransferStats(selectedServer: server.Server, serverView: Polymer) {
-    const refreshTransferStats = () => {
-      selectedServer.getDataUsage().then(
-          (stats) => {
-            // Calculate total bytes transferred.
-            let totalBytes = 0;
-            // tslint:disable-next-line:forin
-            for (const accessKeyId in stats.bytesTransferredByUserId) {
-              totalBytes += stats.bytesTransferredByUserId[accessKeyId];
-            }
-            serverView.setServerTransferredData(totalBytes);
-            // tslint:disable-next-line:forin
-            for (const accessKeyId in stats.bytesTransferredByUserId) {
-              const transferredBytes = stats.bytesTransferredByUserId[accessKeyId];
-              const relativeTraffic = totalBytes ? 100 * transferredBytes / totalBytes : 0;
-              serverView.updateAccessKeyRow(accessKeyId, {transferredBytes, relativeTraffic});
-            }
-          },
-          (e) => {
-            // Since failures are invisible to users we generally want exceptions here to bubble
-            // up and trigger a Sentry report. The exception is network errors, about which we can't
-            // do much (note: ShadowboxServer generates a breadcrumb for failures regardless which
-            // will show up when someone explicitly submits feedback).
-            if (e instanceof errors.ServerApiError && e.isNetworkError()) {
-              return;
-            }
-            throw e;
-          });
-    };
-    refreshTransferStats();
+  private async refreshTransferStats(selectedServer: server.Server, serverView: Polymer) {
+    try {
+      const stats = await selectedServer.getDataUsage();
+      let totalBytes = 0;
+      // tslint:disable-next-line:forin
+      for (const accessKeyId in stats.bytesTransferredByUserId) {
+        totalBytes += stats.bytesTransferredByUserId[accessKeyId];
+      }
+      serverView.setServerTransferredData(totalBytes);
 
+      const accessKeyDataLimit = selectedServer.getAccessKeyDataLimit();
+      if (accessKeyDataLimit) {
+        // Make access key data usage relative to the data limit.
+        totalBytes = accessKeyDataLimit.bytes;
```

Comment:
Is there a reason we're re-using this variable?  Maybe something like `const dataLimit = accessKeyDataLimit && accessKeyDataLimit.bytes`